stdlib/builtins.pyi

```diff
@@ -1066,35 +1066,35 @@ def getattr(__o: object, __name: str, __default: None) -> Any | None: ...
 def getattr(__o: object, __name: str, __default: bool) -> Any | bool: ...
 @overload
 def getattr(__o: object, __name: str, __default: _T) -> Any | _T: ...
-def globals() -> dict[str, Any]: ...
+def globals() -> dict[str, object]: ...
 def hasattr(__obj: object, __name: str) -> bool: ...
 def hash(__obj: object) -> int: ...
 def help(*args: Any, **kwds: Any) -> None: ...
 def hex(__number: int | SupportsIndex) -> str: ...
 def id(__obj: object) -> int: ...
-def input(__prompt: Any = ...) -> str: ...
+def input(__prompt: object = ...) -> str: ...
 @overload
 def iter(__iterable: Iterable[_T]) -> Iterator[_T]: ...
 @overload
 def iter(__function: Callable[[], _T | None], __sentinel: None) -> Iterator[_T]: ...
 @overload
-def iter(__function: Callable[[], _T], __sentinel: Any) -> Iterator[_T]: ...
+def iter(__function: Callable[[], _T], __sentinel: object) -> Iterator[_T]: ...
```

Comment:
Probably outside the scope of this PR, but it seems to me that we can replace the last two overloads with just:
```suggestion
def iter(__function: Callable[[], _T | _U], __sentinel: _U) -> Iterator[_T]: ...
```

Reply:
Not going to add that to this PR; but, I see your point :)